app/views/chargeback_rate/_rates_tabs.html.haml

```diff
@@ -2,7 +2,7 @@
   - if x_node == "root"
     = render :partial => "rates_list"
   - elsif @edit && @edit[:new] && ((params[:pressed] && params[:pressed] != "chargeback_rates_delete") || params[:button] == "reset")
-    = render :partial => "edit"
+    = render :partial => "form"
```

Comment:
@himdel this view can be deletd, it is not being used

Reply:
Ah, perfect, I'll remove it.

Do you mean rates_tabs or the rates form as well?

Reply:
Updated :)